ChatDemo-UI3.0/ChatDemo-UI3.0/Category/UIColor+Hyphenate.m

```diff
@@ -0,0 +1,59 @@
+//
+//  UIColor+Hyphenate.m
+//  ChatDemo-UI3.0
+//
+//  Created by Jerry Wu on 6/3/16.
+//  Copyright © 2016 Jerry Wu. All rights reserved.
+//
+
+#import "UIColor+Hyphenate.h"
+
+@implementation UIColor (Hyphenate)
+
++ (UIColor *)HIColorGreenMajor
+{
+    return [[self class] colorFromHexString:@"#b2dd4c"];
+}
+
++ (UIColor *)HIColorGreenDark
```

Comment:
you're completely right. but I think we can do it for the next round of refactoring.